### DISCUSSION TITLE
is there a donation channel ?

### DISCUSSION THREAD

**Discussion by souhaiebtar** [Ulauncher/Ulauncher#955] [General]
thank you for all your great work, i really want to contribute but as i know too little about `python`, i want  to donate a little something,
even if i'm not using alot, i really like it and i want to to evolve and maybe become something like `alfred`, is there a donation channel ?

**Comment by friday** [comment]
Thank you for the support @souhaiebtar. We really appreciate that. Ultimately though donations and sponsoring is difficult for projects like this. It usually doesn't add up to much in the end. It takes time to manage funds (how/if it should be split between collaborators/contributors or pay maintenance costs). In addition we don't want the expectations that goes with paid software development.

**Comment by sortafreel** [comment]
The same question, spent a bit of time searching for donation button, would love to support Ukrainian product, but the answer is clear. Good luck with the project, you are doing awesome.

**Reply by gornostal** [reply]
Messages like this are a kind of support, too. Brings my motivation up :)
Currently, there's no donation option. I plan to enable this with the next major release.
